### ISSUE TITLE
PBS professional batch scheduler submission error

### ISSUE THREAD

**Issue by pditommaso** [nextflow-io/nextflow#209]
The following error message is reported by PBS professional 13.1.7 when a job is submitted 

```
qsub: invalid option -- 'd'
qsub: directive error: -d /home/users/academy/nus/gmsv925/PROJECTS/NKTCL/work/80/
```

The problem is that the `qsub` tool not support the `-d` command line option to set the job working directory.

**Comment by pditommaso** [comment]
Included in version `0.21.2`

**Comment by pditommaso** [comment]
FYI @BurtonChia
